File: 1006/1006.rb
a = gets.chomp.to_f.round(1)
b = gets.chomp.to_f.round(1)
c = gets.chomp.to_f.round(1)
media = (a*2 + b*3 + c*5)/10
puts "MEDIA = #{sprintf('%.1f', media)}"

File: 1006/1006.c
#include <stdio.h>

int main() {
  double A, B, C,  p1 = 2, p2 = 3, p3 = 5;

  scanf("%lf %lf %lf", &A, &B, &C);

  printf("MEDIA = %.1lf\n", (A*p1 + B*p2 + C*p3)/(p1 + p2 + p3));

  return 0;
}
